### PR TITLE
[ISSUE #8472]: Dependency Governance, Remove useless dependencies. (#8472)

### PR DESCRIPTION
**Pull request by CherishCai**
issue #8472

## What is the purpose of the change

Dependency Governance, Remove useless dependencies.

## Brief changelog

XX

## Verifying this change
`mvn clean install -Dmaven.test.skip=true -P release-nacos`
Make sure there are no compile errors.
务必已保障没有编译错误。

Follow this checklist to help us incorporate your contribution quickly and easily:

* [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: console/pom.xml [hunk 1]
```diff
@@ -36,10 +36,6 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>nacos-config</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.apache.tomcat.embed</groupId>
-            <artifactId>tomcat-embed-jasper</artifactId>
```
Comment (CherishCai) [inline console/pom.xml:41]: no jsp, all rest api

File: pom.xml [hunk 2]
```diff
@@ -762,24 +752,6 @@
...
             </dependency>
             
-            <dependency>
-                <groupId>commons-logging</groupId>
-                <artifactId>commons-logging</artifactId>
-                <version>${commons-logging.version}</version>
-            </dependency>
-            
-            <dependency>
-                <groupId>commons-dbcp</groupId>
```
Comment (CherishCai) [inline pom.xml:772]: no use apache common dependencies

File: config/pom.xml [hunk 3]
```diff
@@ -74,15 +74,6 @@
             <artifactId>nacos-encryption-plugin</artifactId>
         </dependency>
         
-        <!-- for aop -->
-        <dependency>
```
Comment (KomachiSion) [inline config/pom.xml:78]: In config module, nacos use aspect, remove it may cause usage problem.

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: config/pom.xml (hunk 3 above)
Comment (CherishCai) [reply config/pom.xml:78]: spring-boot-starter-aop is enough, nested dependency aspectjweaver include aspectjrt
```xml
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-aop</artifactId>
        </dependency>
```

image

Comment (CherishCai) [reply config/pom.xml:78]: 我已经把依赖清理的改动，在线下环境部署了，Aop 也没有问题

**Review by KomachiSion — APPROVED** [review]

**Comment by KomachiSion** [comment]
There are some conflicts with others commit.

**Comment by CherishCai** [comment]
> There are some conflicts with others commit.

@KomachiSion pom.xm conflict resolved three times 😭😂

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**